### DISCUSSION TITLE
simple block device to_do()

### DISCUSSION THREAD

**Discussion by syeonshin** [huijeong-kim/ministore#11] [Q&A] [answered]
안녕하세요..!  지금 Main의 simple_fake_device는 to_do()가 없고 코드가 채워져 있는데 혹시 의도된걸까용..! 

제가 rebase를 오래 안해서 그동안 꽤 바뀌었길래 깔끔하게 처음부터 다시 해보려고 했는데 todo가 사라져서 여쭤봅니다..!ㅎㅎ

**Comment by huijeong-kim** [comment]
흐억 왜 들어갔죠?! 지난 머지에서 실수로 넣은 것 같아요. 다음 머지 때 삭제하겠습니다ㅠㅠ

**Comment by dhihm** [comment]
오예 답안지

**Reply by dhihm** [reply]
일단 눈을 감고 각 함수들을 todo!()로 바꾸면 실습은 문제 없이 할 수 있긴 합니다 ㅋㅋㅋ

**Comment by huijeong-kim** [comment, marked as answer]
여기두 테스트 코드 이슈는 Issues에 올려주시거나 직접 PR 해주셔도 좋을 것 같습니다...!! 감사함당! :) 이번껀 직접 고치겠습니다~~ㅎㅎ
